LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 ARK Builders
+Copyright (c) 2024 ARK Builders
```

Comment:
Should it be the latest or the earliest year of activity of the license?

Reply:
I guess the earliest year of activity is fine - https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year